### PR TITLE
sysbench load test case

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
sysbench load test case

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
@c4pt0r @ngaut @siddontang @shenli  PTAL

**Comment by ngaut** [comment]
LGTM

File: sysbench/scripts/run_all.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-if [ $# != 9 ]; then
+if [ $# != 10 ]; then
```
Comment (siddontang) [inline sysbench/scripts/run_all.sh]: this means that we must input 10 args to run this? 
Can we support paring argument and using default value?

**Comment by shenli** [comment]
LGTM

**Pull request merged**